src/graphql/operations/user.ts

```diff
@@ -5,7 +5,16 @@ import { capture } from '@snapshot-labs/snapshot-sentry';
 
 export default async function (parent, args) {
   const id = args.id;
-  const query = `SELECT u.* FROM users u WHERE id = ? LIMIT 1`;
+  const query = `
+    SELECT
+      u.*,
+      SUM(l.vote_count) as votesCount,
+      SUM(l.proposal_count) as proposalsCount,
+      MAX(l.last_vote) as lastVote
+    FROM users u
+    LEFT JOIN leaderboard l ON BINARY l.user = BINARY u.id
+    WHERE id = ?
+    LIMIT 1`;
```

Comment:
Looks like this take more than 30 secs

Reply:
As you said Binary casting is causing this, we should do something like this
```suggestion
  const query = `
    SELECT
      u.*,
      SUM(l.vote_count) as votesCount,
      SUM(l.proposal_count) as proposalsCount,
      MAX(l.last_vote) as lastVote
    FROM users u
    LEFT JOIN leaderboard l ON l.user = ?
    WHERE id = ?
    LIMIT 1`;
```

Reply:
Actually, it takes very less we can just use `l.user = BINARY u.id` instead of `BINARY l.user = BINARY u.id`

Reply:
```suggestion
  const query = `
    SELECT
      u.*,
      SUM(l.vote_count) as votesCount,
      SUM(l.proposal_count) as proposalsCount,
      MAX(l.last_vote) as lastVote
    FROM users u
    LEFT JOIN leaderboard l ON l.user = BINARY u.id
    WHERE id = ?
    LIMIT 1`;
```